J/JuliaInterpreter/Compat.toml

```diff
@@ -1,20 +1,39 @@
-["0-0.7"]
-julia = "1"
+["0 - 0.7"]
+julia = "1.0"
+# assuming exact match was wanted
+# otherwise, use ~ or ^ specifier
 
-["0.2-0.7.13"]
-CodeTracking = "0.3.2-0.5"
+["0.2 - 0.7.13"]
+CodeTracking = "0.3.2 - 0.5"
+# note this allows
+# CodeTracking [0.3.2 - 0.6)
+# otherwise use
+# CodeTracking = "0.3.2 - 0.5.8"
 
-["0.7.14-0.7.20"]
-CodeTracking = "0.5.9-0.5"
+["0.7.14 - 0.7.20"]
+CodeTracking = "0.5.9 - 0.5"
 
-["0.7.21-0"]
-CodeTracking = ["0.5.9-0.5", "1"]
+["0.7.21 - 0"]
+CodeTracking = "0.5.9 - 0.5-1"
+# unclear what is meant here
+# or if this is a mistake
+# CodeTracking 0.5.9 to 1 or to 0.5.10?
+# CodeTracking latest version is now at 1.0.8
```

Comment:
```suggestion
["0.7.21-0"]
CodeTracking = ["0.5.9-0.5", "1"]
```

Reply:
I must be missing something. If ["0.5.9-0.5, "1"] means a union of the versions specified, why not just "0.5.9-1"?

Reply:
Because `["0.5.9-0.5, "1"]` doesn't include v0.6 or the other infinite many breaking releases between 0.6 and 1.0`. `"0.5.9-1"` does it instead.